setup.py

```diff
@@ -46,7 +46,7 @@
     tests_require=["pytest<=4.4.0", "nose", "mock", "coverage", "flake8"],
     extras_require={
         'krbV': ["python-krbV"],
-        'beaker': ['beaker-client>=23.3', 'python-krbV'],
+        'beaker': ['beaker-client>=27.0', 'python-gssapi', 'bottle'],
```

Comment:
Why do you need `bottle`?
It is definitely not part of [Beaker](https://github.com/beaker-project/beaker) requirements.